script/tool/lib/src/version_check_command.dart

```diff
@@ -175,9 +176,6 @@ class VersionCheckCommand extends PluginCommand {
           continue;
         }
         final Pubspec pubspec = Pubspec.parse(pubspecFile.readAsStringSync());
-        if (pubspec.publishTo == 'none') {
-          continue;
-        }
```

Comment:
this can be handy if a plugin is a WIP state.

Reply:
this makes testing very hard, because in testing, pubspecs are all created with publishTo = none. See: https://github.com/flutter/plugins/blob/master/script/tool/test/util.dart#L154-L156
So this check basically made tests imposable if we want to test with a pubspec that contains versions. 

I was thinking even for WIP plugins, it is still ok to check to see if versions match.

Reply:
Couldn't we change that test safeguard from `none` to some made up pub server that doesn't exist, so it wouldn't clash with real `none` entries?